src/main/java/com/google/cloud/dataproc/templates/hive/README.md

```diff
@@ -11,9 +11,9 @@ HISTORY_SERVER_CLUSTER=<history-server> \
 bin/start.sh \
 --properties=spark.hadoop.hive.metastore.uris=thrift://<hostname-or-ip>:9083 \
 -- --template HIVETOBIGQUERY \
---templateProperty hivetobq.bigquery.location=<required_bigquery destination>
---templateProperty hivetobq.sql=<hive_sql>
---templateProperty hivetobq.append.mode=<Write_Mode> 
+--templateProperty hivetobq.bigquery.location=<required_bigquery destination> \
+--templateProperty hivetobq.sql=<hive_sql> \
+--templateProperty hivetobq.write.mode=<Write_Mode> \
```

Comment:
Do you think it is worth adding a section in the readme an explanation of posible values for this property, e.g. append, errorifexists ...?